tests/test_routes.py

```diff
@@ -32,6 +32,14 @@ def test_homepage(self):
 
         self.assertEqual(self.client.get("/").status_code, 200)
 
+    def test_careers_overview(self):
+        """
+        When given the index URL,
```

Comment:
Maybe this should be changed to /careers index URL? otherwise this comment is the same as the main `index.html` test